--- ./Makefile.org	Sat Jun 21 07:38:54 2003
+++ ./Makefile	Wed Aug  6 01:16:31 2003
@@ -11,8 +11,9 @@
 CC	= gcc
 LD	= $(CC)
 RM	= rm -f
-CFLAGS	= -O2 -Wall
-LDFLAGS = $(CFLAGS)
+FLAGS	= -O2 -Wall
+CFLAGS	= $(FLAGS)
+LDFLAGS = $(FLAGS)
 DEF     = -I. -I$(includedir)
 LIBPCI  = -lpci
 LDLIBS	= $(LIBPCI)
@@ -28,8 +29,8 @@
 	$(CC) $(CFLAGS) $(OBJS) $(LDLIBS) -o $@
 
 install::
-	install -d $(sbindir)
-	install -s $(PACKAGE) $(sbindir)
+	install -d $(DESTDIR)$(sbindir)
+	install -s $(PACKAGE) $(DESTDIR)$(sbindir)
 
 .c.o:
 	$(CC) $(CFLAGS) $(DEF) -c $< -o $@
